README.Rmd

```diff
@@ -18,37 +18,229 @@ knitr::opts_chunk$set(
+```
+
+
+### Step 2: Overlay administrative regions onto the your data's grid
+A core part of `stagg`'s functionality is to aggregate gridded data to the level of administrative regions. In order to do this, it first calculates the  portion of each region is covered by a particular cell, or the weight each grid cell has for a given administrative region. These weights may also be scaled by the secondary weights calcualted in Step 1. This is accomplished using the `overlay_weights()` function. 
+
+```{r, echo = TRUE, message = FALSE, results = FALSE}
+ # Using polygons outlining counties of Kansas as administrative regions
+kansas_counties <- tigris::counties("Kansas")
 ```
 
-You'll still need to render `README.Rmd` regularly, to keep `README.md` up-to-date. `devtools::build_readme()` is handy for this. You could also use GitHub Actions to re-render `README.Rmd` every time you push. An example workflow can be found here: <https://github.com/r-lib/actions/tree/v1/examples>.
 
-You can also embed plots, for example:
 
-```{r pressure, echo = FALSE}
+```{r, include = TRUE}
+
+county_weights <- overlay_weights(
+  
+  polygons = kansas_counties,          # A simple features object with the 
+                                       # desired polygons
+  
+  polygon_id_col = "COUNTYFP",         # The name of the column containing 
+                                       # polygons' identifiers
+  
+  grid = era5_grid,                    # A raster layer with the same coordinate
+                                       # system and spatial resolution as the 
+                                       # climate_data (defaults to the 
+                                       # era5_grid). You can also pass in your 
+                                       # climate data and a grid will be taken 
+                                       # from its first layer
+  
+  secondary_weights = cropland_weights # Optional output from Step 1, a table of
+                                       # weights, determined here by cropland in
+                                       # each grid cell
+)
+
+# Display results
+county_weights
 ```
 
-In that case, don't forget to commit and push the resulting figure files, so they display on GitHub and CRAN.
+You can see that the function outputs multiple rows for each polygon, one for every grid cell it overlaps. The column `w_area` represents the proportion of that polygon that falls within the grid cell corresponding to the x and y coordinates. If you included secondary_weights from Step 1, as we have here, `overlay_weights()` also creates a column named `weight`, which is determined by normalizing the secondary_weights by `w_area`. This is what will be used in the aggregation of values. If you wish only to use `w_area` in aggregating by polygon, you need not run `secondary_weights()` and can omit the argument `secondary_weights` from your call to `overlay_weights()`. Once again, note that the function is 
```

Comment:
I think the final sentence in this paragraph is missing the ending - "note that the function is ... "